rotkehlchen/tests/unit/test_avalanche_manager.py

```diff
@@ -1,11 +1,29 @@
+from unittest.mock import patch
+
+from flaky import flaky
+
+from rotkehlchen.types import deserialize_evm_tx_hash
+
+
+@flaky(max_runs=5, min_passes=1)  # covalent api might be really flaky
```

Comment:
Is that the case with the change you are doing?

Reply:
Didn't get the question. But in general Covalent API was failing a lot while I was trying to test locally, so I thought to put a `@flaky` decorator. Or is not a good idea?

Reply:
if covalent is flaky it's fine, but please always put max at 3.

Reply:
ok